### ISSUE TITLE
Stopwatch timer not match to category

### ISSUE THREAD

**Issue by mrkeuz** [ActivityWatch/activitywatch#566]
- [x] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version.
 - [x] I have searched the issues of this repo and believe that this is not a duplicate.

- **OS name and version**: Ubuntu 20.04.2 + Chrome 88.0.4324.150 (latest)
- **ActivityWatch version**: Version: v0.10.0

## Describe the bug

Category not match to stopwatch timer, so time report show this time as Uncategorized

## To Reproduce

    1. Add Stopwatch timer "fl-projects" 
    2. Edit timer (set duration i.e. 3 hours)
    3. Add category `Work → Programming → Freelance` with regex "fl-projects"
    4. Open activity time 
    5. Report show time as Uncategorized

## Expected behavior

Report with Categorized time as `Work → Programming → Freelance`

## Documentation
N/A
## Additional context
* I tried restart `aw`
* I tried restart refresh / restart browser 
* I have check database data - `eventmodel` has valid value, label and duration. Start time not intersect with "Start of day" 
* Some data collected with other watchers matches as expected 
* I can provide some additional info if needed

**Comment by mrkeuz** [comment]
Do I understand correctly that it is just not yet supported and `aw-stopwatch` should filter somewhere here for categorization: https://github.com/ActivityWatch/aw-webui/blob/master/src/queries.ts

**Comment by thesophiaxu** [comment]
Strongly seconding this - for me it is an important missing functionality as well. If no one else has time to implement this I'm also happy to work with it :)

**Comment by johan-bjareholt** [comment]
The thing is that categories are only queried on windows, adding stopwatch time into those summaries could create confusion such as total time going above 24 hours in a day.

There is a lot missing regarding the stopwatch feature, such as pretty much any visualization except for just listing each entry. A really simple thing we could start out with would be to add a simple visualization that sums all timers with the same label.

**Comment by mrkeuz** [comment]
Let me explain my idea in a little more detail. Great use case that I see in categorization times Is to mark short periods in time when I.e. go to meeting, googling something, just away for read book. Currently, this time will be uncategorized. But with timers I just create some event manually for past period (for matching exactly for I want)

In this case all my daily uncategorized event just gone. As a solution for long timers (24 hours as you mention) I think it has option (but I'm not sure what is the best) 

* you can just split timer for end of day. 
* split timers more than 24 hours
* automatic stop it after some timeout

In summary, timer matching cool exactly for uncategorized time. So you can analyze time that you cannot easy to log or categorized with simple "Rules". Or you just does not have events for this period.

**Comment by johan-bjareholt** [comment]
> As a solution for long timers (24 hours as you mention) I think it has option (but I'm not sure what is the best)

You are misunderstanding me, I'm not refering to 24hr timers.

Another example, let's say that you start a timer for a 1h meeting and during that meeting maybe present something for 30min. If you add both of those activities your total amount of activity would be 1,5hrs even though only 1hr passed.

That's why I'm proposing a completely different view for visualizations for timers specifically, the same way we do it for editor watchers today.

**Comment by Springvar** [comment]
It would still be very useful to have some kind of support for afk tracking for things like meetings etc. where I don't typically use my computer at all, but still want to have the time tracked.

I realize that this is not the intention of the current stopwatch implementation, but with some modification it could serve this purpose as well. A checkbox when starting the stopwatch could tell it to override any other activity for the duration, or just "fill in the gaps" as some sort of background activity when no other activity is recorded.

**Comment by ErikBjare** [comment]
I think this could be added to the query & visualizations on the Activity
view without too much effort. Double-counting of time could be resolved by
making a period_union (or whatever the transform is called that combines
event lists without overlap, with the first one taking priority). An option
could be added to make the automated tracking or the manual tracking take
priority.
